(#64) Update analytics for Disease

alerosad opened the pull request:
* added click event test to DiseaseAnalytics.feature
Closes #64

alerosad commented:
not sure why the pa11y tests aren't passing, they pass fine on my local machine

alerosad commented:
pa11y tests passing now, not sure why yesterday they were acting up

arcepaul commented on cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature:
```diff
@@ -22,3 +22,18 @@ Feature: Disease listing page analytics
 			| page.additionalDetails.diseaseName          | breast cancer                                             |
 			| page.additionalDetails.trialListingPageType | disease                                                   |
 			| page.additionalDetails.numberResults        | (int)938                                                  |
+
```
As discussed add the line when rebasing.

alerosad replied:
done

sarinapadilla replied:
Actually, this line needs to be added to the below test, with the click event - it's already been added to the first test with the previous commit to develop.

alerosad commented:
it won't pass the DiseaseAnalytics.feature integration test, since I added: 
And "dynamicListingPatterns" object is set to "Disease"
to it. I added this since in Sarina's and Kate's PR they add this new parameter to their tests.

sarinapadilla commented on cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature:
```diff
@@ -22,3 +22,19 @@ Feature: Disease listing page analytics
+	Scenario: Click event fires when a user clicks on result item
+        Given "trialListingPageType" is set to "Disease"
+				And "dynamicListingPatterns" object is set to "Disease"
+       	And "pageTitle" is set to "{{disease_label}} Clinical Trials"
```
Please remove this line, as setting the dynamicListingPatterns on the line above takes care of it. pageTitle being set is only necessary for Manual pages now.
